python/server.py

```diff
@@ -3,13 +3,41 @@
 from pydantic import BaseModel
 
 from models import *
+from external_models import *
 
 app = FastAPI()
 
+
+openai_generation_kwargs = {"model": "gpt-4-turbo-preview",
+                     "temperature": 0.9,
+                     "max_tokens": 1024,
+                     "top_p": 0.9,
+                     "frequency_penalty": 0,
+                     "presence_penalty": 0,
+                     "num_return_sequences": 16,
+                     "openai_timeout": 45,
+                     }
+vllm_generation_kwargs = {"model": "internlm/internlm2-math-plus-1_8b",
+                         "tensor_parallel_size": 2,
+                         "temperature": 0.6,
+                         "max_tokens": 1024,
+                         "top_p": 0.9,
+                         "length_penalty": 0,
+                         "n": 32,
+                         "do_sample": True,
+                         "output_scores": True,
+                         "output_logits": False,
+                         "return_dict_in_generate": True,
+                         "device": "auto",
+                         }
+
 models = {
     # "EleutherAI/llemma_7b": DecoderOnlyTransformer(
     #    "EleutherAI/llemma_7b", num_return_sequences=2, max_length=64, device="auto"
-    #),
+    # ),
+
+    "gpt4": OpenAIRunner(**openai_generation_kwargs),
+    "InternLM":VLLMTacticGenerator(**vllm_generation_kwargs),
     "wellecks/llmstep-mathlib4-pythia2.8b": PythiaTacticGenerator(
         num_return_sequences=32, max_length=1024, device="auto"
```

Comment:
Although there are not many arguments passed for this model, let's make the style consistent and use a parameter list like the ones above. Same for the ones below.